app/Http/Controllers/PayloadHistoryController.php

```diff
@@ -93,4 +93,61 @@ public function destroy(Request $request, PayloadHistory $history)
             ]);
         }
     }
+
+    public function recheck($id) {
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Functions.MultiLineFunctionDeclaration.BraceOnSameLine</br>**Severity**: error</br>**File**: app/Http/Controllers/PayloadHistoryController.php</br>**Line**: 97</br>Opening brace should be on a new line